### DISCUSSION TITLE
Side bar for non-admin users

### DISCUSSION THREAD

**Discussion by gearoidc** [directus/directus#6773] [Archive]
Hi,

Just setup Directus, and was wondering how can you enable the sidebar for non-admins so they can click to files / collections etc, from the screen shot you can see that admin type nav icons are not there. This user does have access to files and I can access them when I enter in the url.

<img width="146" alt="Screen Shot on 2021-07-12 at 21-17-03" src="https://user-images.githubusercontent.com/65091/125350348-8c91f380-e356-11eb-82fb-90119446e3bb.png">

Thanks in advance
--g

**Comment by philleepflorence** [comment]
Hello, hope this helps answer your question on the Side Bar a.k.a Module Bar or Navigation:
> Customizing the Module Navigation is done within the Role's configuration
- Learn about the Module Bar in the [The Directus Application - Concept Docs](https://docs.directus.io/concepts/application/)
- Learn more about [Customizing the Module Navigation - Guide Docs](https://docs.directus.io/guides/roles/#customizing-the-module-navigation)

**Reply by gearoidc** [reply]
Thanks @philleepflorence I'll have a read today

**Reply by agusibrahim** [reply]
> * Customizing the Module Navigation - Guide Docs

[Customizing the Module Navigation - Guide Docs](https://docs.directus.io/guides/roles/#customizing-the-module-navigation) not available now. there is changes about customize module nav mechanism?

**Reply by rijkvanzanten** [reply]
@agusibrahim You can configure the module sidebar from the project settings. Changing it per role/user isn't available

**Reply by agusibrahim** [reply]
@rijkvanzanten how about hide/show collection menu per role without changing read permission?

**Reply by rijkvanzanten** [reply]
Same thing, the navigation is configured globally 👍🏻
